runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -43,21 +43,25 @@ class LayoutFFITypeHelpers
 public:
 	LayoutFFITypeHelpers(J9VMThread *currentThread)
 			: _currentThread(currentThread)
-			, _vm(_currentThread->javaVM)
-	{ };
+			, _vm(currentThread->javaVM)
+	{
+	}
 
 	/**
 	 * @brief Convert the preceding integer of a layout string from string to UDATA,
-	 * and put layout at the position after the integer
+	 * and put layout at the position after the integer.
+	 *
+	 * @param layout[in/out] A pointer to a C string describing the types of the struct elements,
+	 * which points to the symbol of the struct/array upon return after computing the digits.
 	 *
-	 * @param layout[in] A pointer to a c string describing the types of the struct elements. For example:
-	 * 		If layout is "16#2[#2[II]#2[II]]", it returns 16 in byte
+	 * For example, if layout is "16#2[#2[II]#2[II]]", it returns 16 (bytes) which represents
```

Comment:
I think `(bytes)` is redundant in this sentence.

Reply:
Removed.